pandas/_libs/reduction.pyx

```diff
@@ -28,6 +28,14 @@ cdef _get_result_array(object obj, Py_ssize_t size, Py_ssize_t cnt):
     return np.empty(size, dtype='O')
 
 
+cdef bint _is_sparse_array(object obj):
```

Comment:
Would this be better-suited for `pandas._libs.util`? Or keep here since this is the only file using it and it's temporary?

Reply:
Yes, exactly for those reasons (It's only used here, and should be removed again once we get rid of this deprecation), I would keep it here

Reply:
this is not the right location not should be in util
your argument is not correct ; just because we eventually will remove it does not mean it should. it be with similar code